scripts/generate_release_post.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+VERSION="$1" # such as 5.3.0-rc.1, 5.0.0
+LAST_VERSION="$(git for-each-ref --sort=creatordate --format '%(refname)' refs/tags | grep "v[0-9]\.[0-9]\.[0-9]" | grep -v mas | tail -2 | head -1 | sed "s/refs\/tags\/v//")"
+
+cat <<-MD
+### [v$VERSION](https://github.com/mattermost/desktop/releases/tag/v$VERSION) :tada:
+Changes:
+$(git cherry -v v$LAST_VERSION | grep ^+ | grep "(#[0-9]\+)" | sed "s/^+\s[a-zA-Z0-9]\+\s/- /" | sed "s/\s(#\([0-9]\+\))$/ [(#\1)](https:\/\/github.com\/mattermost\/desktop\/pull\/\1)/" | sed "s/\[\?MM-\([0-9]\+\)\]\?/[[MM-\1]](https:\/\/mattermost.atlassian.net\/browse\/MM-\1)/")
+
+The release will be available on GitHub shortly.
```

Comment:
Do we want to link it like before: 

``the new release can be found on [GitHub](https://github.com/mattermost/desktop/releases)."``?

Reply:
We have the link above when you click on the server version. I think we're good with one or the other. I just have the blurb there that if we do a release I still have to manually publish it, so it won't be instantly on GitHub.